gateway-provider-security-authz-composite/src/main/java/org/apache/knox/gateway/deploy/impl/CompositeAuthzDeploymentContributor.java

```diff
@@ -56,17 +56,23 @@ public void contributeFilter( DeploymentContext context, Provider provider, Serv
 
     Map<String, String> providerParams = provider.getParams();
     String providerNames = providerParams.get("composite.provider.names");
+    if (!providerNames.isEmpty()) {
```

Comment:
I'd use StringUtils over String here because the first one is `null` safe. Even better, I think `parseProviderNames` should return a collection and not a String array.